bounties/other/age-gender-estimation/1/vulnerability.json

```diff
@@ -0,0 +1,59 @@
+{
+    "PackageVulnerabilityID": 1,
+    "DisclosureDate": "2020-09-17",
+    "AffectedVersionRange": "*",
+    "Summary": "Resources Downloaded over Insecure Protocol",
+    "Contributor": {
+        "Discloser": "B3EF",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "other",
+        "Name": "age-gender-estimation",
+        "URL": "https://github.com/yu4u/age-gender-estimation",
+        "Downloads": ""
+    },
+    "CWEs": [
+        {
+            "ID": "CWE-494",
+            "Description": ""
+        }
+    ],
+    "CVSS": {
+        "Version": "3.1",
+        "AV": "N",
+        "AC": "H",
+        "PR": "N",
+        "UI": "R",
+        "S": "C",
+        "C": "L",
+        "I": "L",
+        "A": "N",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "4.7"
+    },
+    "CVEs": [
+        ""
+    ],
+    "Repository": {
+        "URL": "https://github.com/yu4u/age-gender-estimation.git",
+        "Codebase": [
+            "Python"
+        ],
+        "Owner": "yu4u",
+        "Name": "age-gender-estimation",
+        "Forks":"",
+        "Stars":""
+    },
+    "Permalinks": [
+        "https://github.com/yu4u/age-gender-estimation/blob/6af6148f6eb09bf16e7164490730c71c61c89f3e/lap/create_lap_dataset.py#L74",

```

Comment:
Always use `permalinks` to files in the `master `branch like https://github.com/yu4u/age-gender-estimation/blob/master/lap/create_lap_dataset.py#L74